example/README.rst

```diff
@@ -46,3 +46,13 @@ environment variable::
 
     $ DB_BACKEND=postgresql python example/manage.py migrate
     $ DB_BACKEND=postgresql python example/manage.py runserver
+
+Using an asynchronous (ASGI) server:
+
+Install [Daphne](https://pypi.org/project/daphne/) first:
+
+    $ python -m pip install daphne
```

Comment:
Let's just add it to `requirements-dev.txt` instead.

Reply:
Since this doc also includes the instruction to install django, I think you were right initially. We should keep this instruction. I did add it to the requirements_dev.txt file though since contributors are likely to run this and it makes sense to simply that flow.